### ISSUE TITLE
[Mandrel 22.3.1] Qpid JMS client native image compilation failure

### ISSUE THREAD

**Issue by jamesnetherton** [apache/camel-quarkus#4466]
All of the integration tests that use the Qpid JMS client fail in native mode.

Nightly build over at the Qpid JMS project is also failing for the same reason:

https://github.com/amqphub/quarkus-qpid-jms/actions/runs/4060641331

```
2023-02-01T04:38:17.9268656Z Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing org.apache.qpid.jms.tracing.opentracing.OpenTracingTracerFactory.createTracer(java.net.URI, java.lang.String) 
2023-02-01T04:38:17.9269386Z Parsing context:
2023-02-01T04:38:17.9270720Z    at org.apache.qpid.jms.tracing.opentracing.OpenTracingTracerFactory.createTracer(OpenTracingTracerFactory.java:72)
2023-02-01T04:38:17.9271457Z    at org.apache.qpid.jms.tracing.JmsTracerFactory.create(JmsTracerFactory.java:47)
2023-02-01T04:38:17.9272071Z    at org.apache.qpid.jms.JmsConnectionFactory.configureConnectionInfo(JmsConnectionFactory.java:282)
2023-02-01T04:38:17.9272728Z    at org.apache.qpid.jms.JmsConnectionFactory.createConnection(JmsConnectionFactory.java:208)
2023-02-01T04:38:17.9273347Z    at org.apache.qpid.jms.JmsConnectionFactory.createConnection(JmsConnectionFactory.java:200)
2023-02-01T04:38:17.9273950Z    at org.springframework.jms.support.JmsAccessor.createConnection(JmsAccessor.java:196)
2023-02-01T04:38:17.9274729Z    at org.springframework.jms.listener.AbstractJmsListeningContainer.createSharedConnection(AbstractJmsListeningContainer.java:412)
2023-02-01T04:38:17.9276455Z    at org.springframework.jms.listener.AbstractJmsListeningContainer.establishSharedConnection(AbstractJmsListeningContainer.java:380)
2023-02-01T04:38:17.9277697Z    at org.springframework.jms.listener.AbstractJmsListeningContainer.doStart(AbstractJmsListeningContainer.java:292)
2023-02-01T04:38:17.9278492Z    at org.springframework.jms.listener.SimpleMessageListenerContainer.doStart(SimpleMessageListenerContainer.java:232)
2023-02-01T04:38:17.9279282Z    at org.springframework.jms.listener.AbstractJmsListeningContainer.start(AbstractJmsListeningContainer.java:277)
2023-02-01T04:38:17.9280287Z    at org.apache.camel.component.jms.JmsConsumer.startListenerContainer(JmsConsumer.java:115)
2023-02-01T04:38:17.9281040Z    at org.apache.camel.component.jms.JmsConsumer.prepareAndStartListenerContainer(JmsConsumer.java:188)
2023-02-01T04:38:17.9281656Z    at org.apache.camel.component.jms.JmsConsumer.doStart(JmsConsumer.java:172)
2023-02-01T04:38:17.9283099Z    at org.apache.camel.support.service.BaseService.start(BaseService.java:119)
2023-02-01T04:38:17.9283629Z    at org.apache.camel.quarkus.main.CamelMain.startEngine(CamelMain.java:140)
2023-02-01T04:38:17.9284459Z    at org.apache.camel.quarkus.main.CamelMain.runEngine(CamelMain.java:150)
2023-02-01T04:38:17.9285107Z    at org.apache.camel.quarkus.main.CamelMainRuntime.lambda$start$0(CamelMainRuntime.java:53)
2023-02-01T04:38:17.9285731Z    at org.apache.camel.quarkus.main.CamelMainRuntime$$Lambda$1063/0x00000007c1b0ef10.run(Unknown Source)
2023-02-01T04:38:17.9286162Z    at java.lang.Thread.run(Thread.java:833)
2023-02-01T04:38:17.9286491Z    at java.lang.Shutdown.runHooks(Shutdown.java:130)
2023-02-01T04:38:17.9286854Z    at java.lang.Shutdown.shutdown(Shutdown.java:185)
2023-02-01T04:38:17.9287295Z    at com.oracle.svm.core.jdk.RuntimeSupport.shutdown(RuntimeSupport.java:154)
2023-02-01T04:38:17.9287583Z 
```

**Comment by jamesnetherton** [comment]
FYI @gemmellr.

**Comment by gemmellr** [comment]
Thanks, I'm aware but spent yesterday dealing with _other_ quarkus stuff and didnt get to look. The CI is still working with quarkus 2.16 so its seems a change on quarkus main has had an effect.

**Comment by gemmellr** [comment]
@jamesnetherton I have released an updated extension, 0.42.0, to handle the new behaviour when using Quarkus main.

**Comment by jamesnetherton** [comment]
Seems everything is working fine since upgrading to `2.0.0.Alpha3`.
